Sources/Phakchi/Encodable.swift

```diff
@@ -1,52 +1,63 @@
 import Foundation
 
-typealias JSONObject = [String: AnyObject]
+typealias JSONObject = [String: PactEncodable]
 
 public protocol PactEncodable {
-    var pactJSON: AnyObject { get }
+    var pactJSON: PactEncodable { get }
```

Comment:
I don't think the return type of `pactJSON` should be `PactEncodable`. You need to have a different type for that. It should not be able to return an `Interaction` or a `Request`.